skills/git-workflow/pr-and-comments.md

```
@@ -102,12 +102,18 @@ Auth fails → ask user to run `gh auth login`.
 2. Read all relevant comments before editing.
 3. Validate each against current code. Skip stale/invalid with evidence.
 4. Triage: blocking, suggestion, question, nit, praise.
-5. Architecture smell (missing boundary/owner/contract) → stop normal fix loop, tell user.
+5. Architecture smell (missing boundary/owner/contract) → investigate if arch smell is real:
+  1. If valid, then stop normal fix loop, tell user.
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)